tests/e2e/default/tasks/tasks.wdio-spec.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path');
+const chtConfUtils = require('@utils/cht-conf');
+const utils = require('@utils');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const tasksPage = require('@page-objects/default/tasks/tasks.wdio.page');
+const { expect } = require('chai');
+
+const places = [
+  {
+    _id: 'fixture:district',
+    type: 'district_hospital',
+    name: 'District',
+    place_id: 'district',
+    reported_date: new Date().getTime(),
+  },
+  {
+    _id: 'fixture:center',
+    type: 'health_center',
+    name: 'Health Center',
+    parent: { _id: 'fixture:district' },
+    place_id: 'health_center',
+    reported_date: new Date().getTime(),
+  },
+];
+
+const clinics = [
+  {
+    _id: 'fixture:scientists',
+    type: 'clinic',
+    name: 'Scientists',
+    parent: { _id: 'fixture:center', parent: { _id: 'fixture:district' } },
+    place_id: 'scientists',
+    reported_date: new Date().getTime(),
+  }
+];
+
+const people = [
+  {
+    _id: 'fixture:einstein',
+    name: 'Albert Einstenin',
+    type: 'person',
+    patient_id: 'einstein',
+    parent: { _id: 'fixture:scientists', parent: { _id: 'fixture:center', parent: { _id: 'fixture:district' } } },
+    reported_date: new Date().getTime(),
+  }
+];
+
+const chw = {
+  username: 'bob',
+  password: 'medic.123',
+  place: 'fixture:center',
+  contact: { _id: 'fixture:user:bob', name: 'Bob' },
+  roles: ['chw'],
+  known: true,
+};
```

Comment:
Please use factories to create places, people, and users.

An example that you can use is the [send-message.wdio-spec.js ](https://github.com/medic/cht-core/blob/master/tests/e2e/default/sms/send-message.wdio-spec.js) test file, in there we are using `placeFactory`, `personFactory`, and `userFactory`